include/seqan3/alignment/pairwise/detail/type_traits.hpp

```diff
@@ -24,6 +25,7 @@
 #include <seqan3/alignment/configuration/align_config_vectorise.hpp>
 #include <seqan3/alignment/matrix/trace_directions.hpp>
 #include <seqan3/alignment/pairwise/detail/concept.hpp>
+#include <seqan3/core/detail/empty_type.hpp>
```

Comment:
This include should be under core/algorithm.

Reply:
I am not sure I understand what you mean. It is a generic type that can be used in many different contexts.

Reply:
Sorry for the confusion I meant it in the context of lexicographical order. So first core/algorithm... then core/detail